library/network/src/lib/y2firewall/firewalld/api.rb

```diff
@@ -105,7 +105,7 @@ def disable!
 
       # @return [Boolean] The firewalld service state (exit code)
       def state
-        case Yast::Execute.on_target("firewallctl", "state", allowed_exitstatus: [0, 252])
+        case Yast::Execute.on_target("firewall-cmd", "--state", allowed_exitstatus: [0, 252])
```

Comment:
(I very much like the small change but I have no idea if it is correct. I really should get my own 15.0 for testing.)

Reply:
I have it and tested manually.

Reply:
Will adapt the documentation:

http://www.firewalld.org/documentation/man-pages/firewall-cmd.html

Reply:
![firewalldstate](https://user-images.githubusercontent.com/7056681/40050254-d387bfe2-582e-11e8-82fb-70e32eb329fa.png)